priv/templates/jsonapi/view.ex

```diff
@@ -0,0 +1,19 @@
+defmodule <%= module %>View do
+  use <%= base %>.Web, :view
+
+  use JSONAPI.PhoenixView
+
+  def type, do: "<%= singular %>"
+
+  def attributes(model) do
+    Map.take(model, [:id, :created_at])
```

Comment:
You need to take a look at the phoenix generators here https://github.com/phoenixframework/phoenix/blob/master/lib/mix/tasks/phoenix.gen.model.ex#L56 

The default gen.contrller.json that this is based on calls the model generator too